doc/development/building.rst

```diff
@@ -0,0 +1,16 @@
+Building Applications
+=====================
+
+`Jenkins <https://tssw-ci.lsst.org/>`_ is the chosen Continuous Integration platform.
+Each application should have a build project in Jenkins.
+These applications should then have a dedicated build server.
+
+Each build should run the unit tests.
+If the unit tests pass, the application should generate the deployable package and feed into the deployment system.
+Puppet is the Deployment application.
```

Comment:
This is not strictly correct.. I think only a handful of applications use Puppet. In general we use Puppet to provision and manage the infrastructure and k8s to deploy. 

In any case, this is talking about "Build" so maybe remove this part about deployment.

Reply:
Done